simpleviewinc/apex-prepare-demo
# DMSMVR-2723: Add Summit Data to Prepare Demo

Ticket: [DMSMVR-2723](https://simpleviewtools.atlassian.net/browse/DMSMVR-2723)

For reference, original summit demo source data can be found here:
https://drive.google.com/drive/u/0/folders/1C9lvgCS3wBF9A2kXBkytaaXkeTw0_y10

To test:
1. On local, make sure you are running the latest version of DMS `develop` or rebase the branch you are currently working with. There was an update to DMS-CORE that increased the request payload limit. If you try to run the demo script with an older version, you'll get an error `PayloadTooLargeError: request entity too large`
1. Run the Prepare Demo script with the following settings:
    ```
    {
	    "user": "azjmk",
	    "branch": "DMSMVR-2723-Add-Summit-Data-to-Prepare-Demo"
    }
    ```
1. Confirm record counts for the modules with new data:
    1. Accommodation Facilities: 75
        1. `query accommodation_facilities { dms { models { accommodation_facilities_query { count } } } }`
    2. Inquiries: 159
        1. `query inquiries { dms { models { inquiries_query { count } } } }`
    3. Leisure Events: 138
        1. _Can verify count via Apex UI_
    4. Listings: 122
        1. _Can verify count via Apex UI_
    5. Offers: 73
        1. _Can verify count via Apex UI_
    7. Properties: 122
        1. _Can verify count via Apex UI_
    8. Referrals: 115
        1. `query query { dms { models { referrals_query { count } } } }`

[DMSMVR-2723]: https://simpleviewtools.atlassian.net/browse/DMSMVR-2723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
